include/storage/fbw.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
```

Comment:
```suggestion
 * Copyright (c) 2017, 2020 Nordic Semiconductor ASA
```

Reply:
Likewise elsewhere, I think

Reply:
OK, even though this is a copy-paste with a simple rename of old code? I'm not well known in the world of copyright :)

Reply:
Me neither, but it seems like you're adding various pieces of documentation etc. that are new. Not sure if there's a well defined policy.